airbyte-integrations/connectors/source-stripe/source_stripe/source.py

```diff
@@ -297,7 +297,15 @@ def streams(self, config: MutableMapping[str, Any]) -> List[Stream]:
             # The Refunds stream does not utilize the Events API as it created issues with data loss during the incremental syncs.
             # Therefore, we're using the regular API with the `created` cursor field. A bug has been filed with Stripe.
             # See more at https://github.com/airbytehq/oncall/issues/3090, https://github.com/airbytehq/oncall/issues/3428
-            CreatedCursorIncrementalStripeStream(name=_REFUND_STREAM_NAME, path="refunds", **incremental_args),
+            # Based on https://app.zenhub.com/workspaces/critical-connectors-659c693d906fcd00178900cf/issues/gh/airbytehq/airbyte-internal-issues/2450
+            # We are commenting this out just in case a rollback is needed. We are putting refunds back as UdatedCursorIncremental
+            # CreatedCursorIncrementalStripeStream(name=_REFUND_STREAM_NAME, path="refunds", **incremental_args),
+            UpdatedCursorIncrementalStripeStream(
+                name="refunds",
+                path="refunds",
+                event_types=["refund.created", "refund.updated", "charge.refunded", "charge.refund.updated"],
```

Comment:
`charge.refunded` returns a `charge` object and not a `refund` one. We probably need to add some logic to fetch the field `data[].data.object.refunds[]`. There might be duplication (which I'm not too worried about as people can use dedup) as `data[].data.object.refunds` is a list and I'm not sure if we know which one needs to be emitted as a record